### PR TITLE
Show extra info

### PR DESCRIPTION
**Pull request by alexanderGerbik**
Show extra task information (runway IDs, track IDs) in CVAT.

### CODE REVIEW

File: cvat-ui/src/components/task-page/job-list.tsx [hunk 1]
```diff
@@ -36,6 +36,17 @@ interface Props {
     onJobUpdate(jobInstance: any): void;
 }
 
+const ExtraInfoViewer = ({ data }) => {
+    if (!data) return false;
+    const lines = Object.entries(data).map(([key, value]) => `${key}: ${value}`);
```
Comment (gyakushev) [inline cvat-ui/src/components/task-page/job-list.tsx:41]: nit: it's useful to have separate spans for key and value, you can style them differently then

File: cvat/apps/engine/ddln/tasks/spotter/handler.py [hunk 2]
```diff
@@ -17,6 +20,42 @@ def finalize_task_creation(self, task):
         if hints_dir.exists():
             load_hints(hints_dir, task)
 
+    def get_extra_data(self, task):
```
Comment (gyakushev) [inline cvat/apps/engine/ddln/tasks/spotter/handler.py]: `get_extra_info` ?

File: cvat/apps/engine/ddln/tasks/spotter/handler.py [hunk 3]
```diff
@@ -17,6 +20,42 @@ def finalize_task_creation(self, task):
...
+        task_name = guess_task_name(task.name)
+        scenario_files = list(settings.INCOMING_TASKS_ROOT.joinpath(task_name).glob("spo*.csv"))
+        if len(scenario_files) != 1:
+            return None
+        scenario_file = scenario_files[0]
+        reader = pycsv.reader(scenario_file.open('rt', newline=''), lineterminator="\n")
+        result = {}
```
Comment (gyakushev) [inline cvat/apps/engine/ddln/tasks/spotter/handler.py:30]: `OrderedDict` ?

**Review by gyakushev — APPROVED** [review]

File: cvat/apps/engine/ddln/tasks/spotter/handler.py (hunk 3 above)
Comment (alexanderGerbik) [reply cvat/apps/engine/ddln/tasks/spotter/handler.py:30]: `result` is mapping from `sequence_name` to extra data entry for that sequence, I don't need to establish order here, as the order on the UI is defined by the order  of sequences in the main information. However, `OrderedDict` is needed to define the order of fields in extra data entry.

File: cvat/apps/engine/ddln/tasks/spotter/handler.py (hunk 2 above)
Comment (alexanderGerbik) [reply cvat/apps/engine/ddln/tasks/spotter/handler.py]: Fixed

File: cvat-ui/src/components/task-page/job-list.tsx (hunk 1 above)
Comment (alexanderGerbik) [reply cvat-ui/src/components/task-page/job-list.tsx:41]: Sounds reasonable, I'll add spans when/if I have to add styles to them.

**Pull request merged**